src/utils/contribution.js

```diff
@@ -42,11 +44,38 @@ export default class Contribution {
     return result
   }
 
-  /**
-   * Check if the current component has listed changes
-   *
-   */
-  static hasChange = entry => {
+  // Check if the current component has listed changes
```

Comment:
commenting nit: I'd rather there function and parameter names be good than have to add comments like this. For example, `hasChange` is pretty clear that it is testing for changes. Indeed it is testing the provided argument, not the current component. So if the function were `hasChanges(component)` then it really ought not even need a comment. 

Similar feed back on the comments for the other functions.

Summary being that if we have to comment simple things like this then the naming or structuring could probably be improved.

Reply:
You are totally right, we will do some improvements on this.